.github/workflows/release-docker-github-data-migration.yml

```
@@ -23,6 +23,14 @@
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      # https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      # https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)